pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -206,7 +206,7 @@ def select_accelerator(self):
 
         # ddp script mode uses the same flags as TE
         # TODO: decouple from TE
-        if os.environ.get('PL_DDP_PID', False):
+        if os.environ.get('PL_IN_DDP_SUBPROCESS', False):
```

Comment:
is there documentation around what env vars lightning sets anywhere?

Reply:
No, I am sure there is no documentation about these. They are internally used only, not meant to be modified by user. 
Should they be documented? I'm not sure

Reply:
ah I meant for contributors/developers primarily. I should probably read more of the code too :)

Reply:
ah, good point. where could env variables be documented, they are like global variables. maybe at the top of file of the accelerator base class. I have no good suggestions :)